sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -81,6 +82,7 @@ private CommunicationIdentityClient(string endpoint, HttpPipeline httpPipeline,
         {
             _clientDiagnostics = new ClientDiagnostics(options);
             RestClient = new CommunicationIdentityRestClient(_clientDiagnostics, httpPipeline, endpoint, options.ApiVersion);
+            TurnRestClient = new CommunicationIdentityRestClient(_clientDiagnostics, httpPipeline, endpoint, "2021-12-12");
```

Comment:
Next time that we update the swagger, let's make sure the preview api contains both TURN and Identity routes to avoid having two clients

Reply:
Discussing this offline with Reza. There were some concerns in just copying all of identity routes to a preview version without any effective changes.

Reply:
This will be addressed (based on outcome of discussion) in a subsequent PR.